extensions/cmDotCom/client/client.ts

```diff
@@ -0,0 +1,50 @@
+import api from 'api'
+import { type CmSDK, type MessageRequest } from './types'
+
+export class CmClient {
+  private readonly _client: CmSDK
+  private readonly _productToken: string
+  private readonly _activityId: string
+
+  constructor({
+    productToken,
+    activityId,
+  }: {
+    productToken: string
+    activityId: string
```

Comment:
Is there a specific reason you made `activityId` a property of the class constructor? It assumes we are always in the context of an activity when we create an instance of the class. While maybe true right now, by design that looks a bit odd to me. Especially because you use it for the `reference` field of the `SendMessage` body and therefore it might make more sense to also scope that property to the `sendSms` function.

Suggestion:
```typescript
readonly sendSms = async ({
    from,
    to,
    message,
    reference // optional string prop
  }: MessageRequest): ReturnType<CmSDK['messages_SendMessage']> => {
    return await this._client.messages_SendMessage({
      messages: {
        authentication: { productToken: this._productToken },
        msg: [
          {
            from,
            body: {
              type: 'auto',
              content: message,
            },
            reference,
            ...
          },
        ],
      },
    })
  }
```

Reply:
You are right, it makes no sense